scripts/post_npm_install.mjs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import download from 'download';
+import { access } from 'fs/promises';
+import { constants } from 'fs';
+import path from 'path';
+import url from 'url';
+import { format } from 'util';
+
+const jsignDirectoryPath = path.resolve('build/tmp');
```

Comment:
Let's place the artifact under build/third_party/jsign.
We can use the same structure later for all the deps that are using Carthage.